src/components/screens/form-edit-screen.tsx

```diff
@@ -33,9 +39,11 @@ interface Props extends WithStyles<typeof styles> {
 interface State {
   error?: string | Error | Response;
   loading: boolean;
-  metaform?: Metaform;
-  value:string;
+  metaform: Metaform;
+  value: string;
   readOnly: boolean;
+  sections?: MetaformSection[],
```

Comment:
Didn't see the reason to make it undefinable. could be just
```
  sections: MetaformSection[],
``` 
add init it as []

Reply:
There was no purpose for sections altogether, removed it.